traits/tests/test_callable.py

```diff
@@ -0,0 +1,60 @@
+# (C) Copyright 2005-2020 Enthought, Inc., Austin, TX
+# All rights reserved.
+#
+# This software is provided without warranty under the terms of the BSD
+# license included in LICENSE.txt and may be redistributed only under
+# the conditions described in the aforementioned license. The license
+# is also available online at http://www.enthought.com/licenses/BSD.txt
+#
+# Thanks for using Enthought open source!
+
+import unittest
+
+from traits.api import Callable, HasTraits, TraitError
+
+
+
```

Comment:
nit: too many blank lines

Reply:
Whoops, I am sorry. My IDE seems to have bailed out on linting.

Reply:
> My IDE seems to have bailed out on linting.

Ah... it is setup.cfg.  I see!
(Fixed anyway)